### ISSUE TITLE
Coercing methylKitRawDB to GRange error

### ISSUE THREAD

**Issue by AndrewC160** [al2na/methylKit#226]
Hello,

When attempting to coerce a `methylRawDB` object to `GRanges`, data.table::fread does not ignore the header rows in the flat file and so fails to read the table:

```
> class(x)
[1] "methylRawDB"

> as(x,"GRanges")

Uncompressing file.
Reading file.
**Error in stop_if_wrong_length("'seqnames'", ans_len) : 
  'seqnames' must have the length of the object to construct (1859190) or length 1
In addition: Warning message:
In data.table::fread(file = filepath, ...) :
  Detected 1 column names but the data has 7 columns (i.e. invalid file). Added 6 extra default column names at the end.**
```

For anyone running into this error, the following function can be used to bypass the issue by issuing a `zcat` command through fread:

```
  library(tidyverse)
  library(magrittr)
  library(data.table)
  library(GenomicRanges)

  manual_coerce  <- function(db_in = x){
    fread(cmd = paste("zcat",getDBPath(db_in),"| grep -v '#' "),
          col.names = c("seqnames","start","end","strand","coverage","numCs","numTs")) %>%
      makeGRangesFromDataFrame(keep.extra.columns = TRUE)
  }
```

**Comment by alexg9010** [comment]
Hi @AndrewC160,

thanks for using methylKit. Unfortunately, I am not able to reproduce your error with our test data. 

Would you mind posting a minimal example that reproduces your error and giving us more details about your data and the version of methylKit you are using?

Best, Alex

**Comment by alexg9010** [comment]
This seems to be the same error as reported in https://github.com/al2na/methylKit/issues/223

**Comment by alexg9010** [comment]
Hi @AndrewC160 ,

this should be fixed in version 1.17.5 now, but please let me know if you still have these problems.

**Comment by AndrewC160** [comment]
Hi @alexg9010 ,

Apologies for disappearing, I entered the post-doc interview gauntlet and only now emerged...thanks for the heads up!
